tests/unit/routes/branding-test.js

```diff
@@ -1,11 +1,11 @@
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
 
-module('Unit | Route | logos', function(hooks) {
+module('Unit | Route | branding', function(hooks) {
```

Comment:
we should delete this test because it's empty

Reply:
https://github.com/ember-learn/ember-website/pull/508 ;)